CONTRIBUTING.md

```diff
@@ -1,32 +1,41 @@
 # OSS Toolkit Contributing Guide
 
-Thank you for your interest in contributing to Cohere's OSS toolkit. This guide will help explain the contribution workflow from opening an issue, creating a PR, to reviewing and merging your added changes. For development setup, please refer to the [README](README.md).
+Thank you for your interest in contributing to Cohere's OSS toolkit. This guide covers the contribution workflow, from opening an issue to reviewing and merging changes. For development setup, please refer to the [README](README.md).
```

Comment:
Are these changes the same as #792?

Reply:
Yep thats correct, just merged the other one in